Instant-Messenger/include/util/definitions.hpp

```diff
@@ -38,4 +38,12 @@
 
 #ifndef LEFT_GROUP_MESSAGE
 #define LEFT_GROUP_MESSAGE "<Left the group>"
+#endif
+
+#ifndef CONNECTION_LIVE
+#define CONNECTION_LIVE 10
+#endif
+
+#ifndef CONNECTION_COUNTDOWN_STEP
```

Comment:
Judge me
But would'nt be better smth like `KEEP_ALIVE_INTERVAL` ?

Reply:
Naming is important and you definitely got a better one for this. I'll change